pkg/importer/dataStream.go

```diff
@@ -112,12 +122,12 @@ func NewDataStream(endpt, accKey, secKey string) (*dataStream, error) {
 
 // Read from top-most reader. Note: ReadFull is needed since there may be intermediate,
 // smaller multi-readers in the reader stack, and we need to be able to fill buf.
-func (d dataStream) Read(buf []byte) (int, error) {
+func (d *dataStream) Read(buf []byte) (int, error) {
```

Comment:
DataStream has a [512]byte slice and some other simple fields. So that can be a reason for using a ptr. Also, many of the DataStream methods modify their receiver.

Reply:
My motivation was just for consistency.  I'm a little OCD.